packages/amplication-data-service-generator/src/util/nestjs-code-generation.ts

```diff
@@ -36,3 +37,37 @@ export function addInjectableDependency(
     })
   );
 }
+
+export function removeIdentifierFromUseInterceptorDecorator(
+  node: ASTNode,
+  decoratorName: string,
```

Comment:
the decorator name should be const in the function and not a parameter

Reply:
why? UseInterceptors() can get arguments (identifier[]) and you want that the caller of this function sends the argument that needs to be removed

Reply:
I am not talking about the identifier, I am talking about the decorator name and a not about the identifier